client/components/TestQueueRow/index.jsx

```diff
@@ -120,6 +127,24 @@ const TestQueueRow = ({
                 Published {gitUpdatedDateToString(testPlanVersion.updatedAt)}
             </p>
         );
+
+        const latestTestPlanVersion = latestTestPlanVersions.filter(
+            version => version.latestTestPlanVersion.id === testPlanVersion.id
+        );
+        const updateTestPlanVersionButton =
+            isAdmin && latestTestPlanVersion.length === 0 ? (
+                <Button
+                    className="updater-button"
+                    onClick={() => setShowTestPlanUpdaterModal(true)}
+                    size="sm"
+                    variant="outline-secondary"
+                >
+                    {' '}
+                    Update Test Plan Version
+                </Button>
+            ) : (
+                ''
```

Comment:
nitpick and completely optional: I'm not sure if the same happens with hooks as it did with `React.Component` but doing this could create a `React.Text` node I believe.
It definitely has no impact on performance like this of course but the standard for preventing component rendering also seems to be defaulting to `null`.

Could also just avoid the ternary altogether and do something like:
```jsx
const updateTestPlanVersionButton =
            isAdmin && latestTestPlanVersion.length === 0 && (
                <Component ... />
            );
```